src/dune_rules/foreign.ml

```diff
@@ -103,7 +104,12 @@ module Stubs = struct
         let+ loc, lib_name = sum [ ("lib", located Lib_name.decode) ] in
         Lib (loc, lib_name)
       in
-      parse_lib <|> parse_dir
+      let include_ =
```

Comment:
This feature is introduced in 3.5, therefore it should only be available when 3.5 is enabled. See some example in dune_file.ml on how to do that.

Reply:
Is it correct that if an older version is enabled, dune should behave as it did prior to this feature being added, and not explicitly raise an error indicating the minimum version supporting this feature?

Reply:
Yes, that is correct. However, if previous versions would reject `(:include ..)` as an unexpected list, here we should hint to the user that this feature is available starting from `3.5`.